packages/go_router_builder/lib/src/route_config.dart

```diff
@@ -105,6 +123,42 @@ class RouteConfig {
   final String _path;
   final InterfaceElement _routeDataClass;
   final RouteConfig? _parent;
+  final String? _key;
+  final bool _isShellRoute;
+
+  static String? _generateNavigatorKeyGetterCode(
+    InterfaceElement classElement, {
+    required String keyName,
+  }) {
+    bool whereStatic(FieldElement element) => element.isStatic;
```

Comment:
why create one-off functions for the tiny ones – but keep the big function inline?

Reply:
No strong reason I believe.
Do you want me to make a PR to make everything inline?
Do I need to create an issue for this?

Reply:
No issue needed. just a cleanup PR. I'd put everything inline

Reply:
I grouped everything under one `.where` https://github.com/flutter/packages/pull/3698